docs - troubleshooting guide: load from h2 (#17209)
metabase/metabase: docs/troubleshooting-guide/application-database.md

### Metabase fails to start due to database locks

> Sometimes Metabase will fail to complete its startup due to a database lock that was not cleared properly. 

Reviewer: ```suggestion
Sometimes Metabase will fail to start up because a database lock that was not cleared properly. 
```

Reviewer: What is the symptom of this?
